subsystem/elevator.py

```diff
@@ -155,3 +167,8 @@ def periodic(self) -> None:
         table.putNumber('elevator target height', self.target_length)
         table.putNumber('elevator motor lead applied output', self.motor_extend.motor.getAppliedOutput())
         table.putNumber('elevator motor follow applied output', self.motor_extend_follower.motor.getAppliedOutput())
+        table.putNumber('elevator current', self.motor_extend.motor.getOutputCurrent())
+        
+        # set drivetrain control speed
+        states.drivetrain_controlled_vel = constants.drivetrain_max_vel * max((1 - (self.get_length() / constants.elevator_max_length)), .25)
```

Comment:
why are we dividing by constants.elevator_max_length? What values do you think you are going to get there?